lib/libc/Kconfig

```diff
@@ -16,6 +16,21 @@ config NEWLIB_LIBC
 	  Build with newlib library. The newlib library is expected to be
 	  part of the SDK in this case.
 
+config NEWLIB_LIBC_ALIGNED_HEAP_SIZE
+	int
+	prompt "Newlib aligned heap size"
+	depends on CONFIG_MPU_REQUIRES_POWER_OF_TWO_ALIGNMENT
+	depends on USERSPACE
```

Comment:
Shouldn't this depend on NEWLIB_LIBC as well?  Other configs around this one seems to depend on this as well